styles.css

```diff
@@ -1,111 +1,95 @@
-/* Style for the entire page */
 body {
-    background-color: #f9f9f9;
-    color: #444444;
-    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
-    font-size: 16px;
-    line-height: 1.5;
-    margin: 0;
-    padding: 10px;
-    }
-    
-    /* Style for the page title */
-    h1 {
-    font-size: 48px;
-    font-weight: bold;
-    margin: 40px 0;
-    text-align: center;
-    text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2);
-    }
-    
-    /* Style for the section headings */
-    h2 {
-    font-size: 36px;
-    margin: 40px 0 20px;
-    text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2);
-    }
-    
-    /* Style for the links */
-    a {
-    color: #2b9af3;
-    text-decoration: none;
-    transition: all 0.3s ease;
-    }
-    
-    a:hover {
-    color: #1663b3;
-    text-decoration: underline;
-    }
-    
-    /* Style for the lists */
-    ul {
-    list-style: none;
+    font-family: Arial, sans-serif;
     margin: 0;
     padding: 0;
-    }
-    
-    li {
-    margin: 10px 0;
-    }
-    
-    /* Style for the paragraphs */
-    p {
-    margin: 20px 0;
-    text-align: justify;
-    }
-    
-    /* Style for the code blocks */
-    code {
-    background-color: #f5f5f5;
-    border-radius: 4px;
-    color: #444444;
-    font-family: Consolas, monospace;
-    font-size: 14px;
-    padding: 2px 4px;
-    }
-    
-    pre code {
-    background-color: transparent;
-    border-radius: 0;
-    padding: 0;
-    }
-    
-    /* Style for the GitHub markdown */
-    .markdown-body {
-    box-sizing: border-box;
-    max-width: 980px;
-    margin: 0 auto;
-    padding: 45px;
-    }
-    
-    /* Style for the header link */
-    .markdown-body h2 a {
-    color: #2b9af3;
+    background-color: #f7f7f7; /* Light gray background */
+    font-size: 24px; /* Increased font size for the body */
+  }
+  
+  header {
```

Comment:
I don't this current HTML have header tag. So this won't work in given structure.

Reply:
I will work on it today.